pkg/limits/frontend/http.go

```diff
@@ -12,46 +11,6 @@ import (
 	"github.com/grafana/loki/v3/pkg/util"
 )
 
-const ringStreamUsageTemplate = `
```

Comment:
I decided to remove this as I don't believe we ever used it. If you have strong opinions about keeping it let me know and I will add an `All() map[K, V]` method to the cache to support this HTML page. My preference is to remove it though.